client/signup/steps/creds-permission/index.jsx

```diff
@@ -0,0 +1,91 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import Button from 'components/button';
+import QuerySites from 'components/data/query-sites';
+import SignupActions from 'lib/signup/actions';
+import { autoConfigCredentials } from 'state/jetpack/credentials/actions';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class CredsPermissionStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	autoConfigCredentials = () =>
+		this.props.autoConfigCredentials( this.props.initialContext.query.blogid );
+
+	skipStep = () => this.props.goToNextStep();
+
+	shareCredentials = () => {
+		this.autoConfigCredentials();
+		
+		this.props.recordTracksEvent( 'calypso_pressable_nux_credentials_share', {} );
+		
```

Comment:
You have whitespaces in this line 39 and in line 37 that are making Circle CI fail:

```js
/home/ubuntu/wp-calypso/client/signup/steps/creds-permission/index.jsx
  37:1  error  Trailing spaces not allowed  no-trailing-spaces
  39:1  error  Trailing spaces not allowed  no-trailing-spaces

✖ 2 problems (2 errors, 0 warnings)
```